test/test_browser.py

```diff
@@ -2312,6 +2312,9 @@ def test_openal_error(self, args):
   def test_openal_capture_sanity(self):
     self.btest('openal_capture_sanity.c', expected='0')
 
+  def test_openal_extensions(self):
+    self.btest('openal_extensions.c', expected='0')
```

Comment:
Can you use `best_exit`?  This means you can remove the `REPORT_RESULT` stuff from the test and remove the expected argument here (since the default is zero).